baseapp/baseapp.go

```diff
@@ -34,6 +34,12 @@ type (
 	StoreLoader func(ms storetypes.CommitMultiStore) error
 )
 
+// MigrationModuleManager is the interface that a migration module manager should implement to handle
+// the execution of migration logic during the beginning of a block.
+type MigrationModuleManager interface {
+	RunMigrationBeginBlock(ctx sdk.Context, req abci.RequestBeginBlock) bool
```

Comment:
need the error return?

Reply:
47 will [panic](https://github.com/crypto-org-chain/cosmos-sdk/blob/281eb94ee9159d03c6ac589f1e4664c400d7db41/baseapp/abci.go#L154) if have error

Reply:
yes, why not?

Reply:
I mean unlike [50](https://github.com/cosmos/cosmos-sdk/blob/release/v0.50.x/types/module/module.go#L713), no error return from [BeginBlock](https://github.com/crypto-org-chain/cosmos-sdk/blob/281eb94ee9159d03c6ac589f1e4664c400d7db41/types/module/module.go#L583C5-L583C23) in 47

Reply:
yes, it should panic on error